README.md

```diff
@@ -1,4 +1,41 @@
 # BSC-MEV-Sentry
 
-BSC-MEV-Sentry is a proxy service within the BSC MEV architecture designed to shield the validator network.
-It functions to forward requests and pay to builders.
+BSC-MEV-Sentry serves as the proxy service for BSC MEV architecture, It has the following features:
+
+1. Forward RPC requests under mev namespace to builders.
+2. Pay builders for their bid.
+3. Monitor validators' status and health.
+
+See also: https://github.com/bnb-chain/BEPs/pull/322
+
+# Usage
+
+Sentry settings are configured in the `config.toml` file. The following is an example of a `config.toml` file:
+
+```
+[Service]
+HTTPListenAddr = "localhost:8555"
+RPCConcurrency = 10
+RPCTimeout = "10s"
+
+[Account]
+Mode = "keystore"
+KeystorePath = "./keystore"
+Password = "sentry"
```

Comment:
I think password is not propoer in a config.tom, we should have a different source to configure password. For example,  password is stored in a file, and after the program read the file , the file can be deleted.

Reply:
update to using password file, but can be deleted? how to ensure reboot

Reply:
The init container or prepare script will do the job when reboot.

Reply:
ok